ci-operator/config/openshift/loki/openshift-loki-main.yaml

```diff
@@ -28,18 +28,8 @@ images:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: promtail
 promotion:
-  name: "4.11"
-  namespace: ocp
-releases:
-  initial:
-    integration:
-      name: "4.11"
-      namespace: ocp
-  latest:
-    integration:
-      include_built_images: true
-      name: "4.11"
-      namespace: ocp
+  name: 5.y
+  namespace: logging
```

Comment:
@deads2k i seem to recall you working w/ some people to setup loki for our CI envs, but i'm not sure where that setup is consuming the loki+promtail images from, is this going to cause problems?

Reply:
@petr-muller you may know also (where we consume loki/promtail from for our CI job log gathering functionality)

Reply:
@xperimental i think you need to update the mirroring rules too:

https://github.com/openshift/release/blob/f08ff71c0a0ad68ab885099eff85bc37448ae1f0/core-services/image-mirroring/openshift/mapping_origin_4_11#L191

https://github.com/openshift/release/blob/f08ff71c0a0ad68ab885099eff85bc37448ae1f0/core-services/image-mirroring/openshift/mapping_origin_4_11#L258

(and remove the mirroring rules from the older release files since you appear to be dropping the promotion of those)

Reply:
> It seems like the [last commit of this PR](https://github.com/openshift/release/pull/27245/commits/0b0fc0921f2140f3abe56c4ace911e3b89408181) already does these changes or did I miss a file?

sorry, you're correct.

still need to sort out if stopping promoting/mirroring these images to the old location is going to cause problems for CI usage

Reply:
discussion here:
https://coreos.slack.com/archives/CBN38N3MW/p1648820932352989

Per https://github.com/openshift/release/blob/master/clusters/build-clusters/common_except_app.ci/loki.yaml, CI is using `quay.io/openshift-logging/promtail:v2.2.1` but we haven't yet tracked down what is building/publishing that.  Nor what loki image is being used.